include/osquery/killswitch.h

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <boost/core/noncopyable.hpp>
+#include <osquery/expected.h>
+#include <osquery/status.h>
```

Comment:
I didn't find the usage of Status in the file, do you really need to include it here?

Reply:
refresh uses status. Will get rid of status es in the following commits. 👍